src/js/load-package.ts

```diff
@@ -44,7 +44,7 @@ export async function initializePackageIndex(
     );
   }
 
-  if (lockfile.info.version !== API.version) {
+  if (lockfile.info.version !== API.version && false) {
```

Comment:
I guess this need to be reverted before merge?

Reply:
I don't think I meant to commit this at all but yes.